src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -252,41 +268,42 @@ def resolve_dependencies(self, main_ctx: AnalyzeContext, sbom_path: str) -> dict
             return {}
 
         # Grab dependencies for each build tool, collate all into the deps_resolved
-        for tool in build_tools:
+        for build_tool in build_tools:
             try:
-                dep_analyzer = tool.get_dep_analyzer(main_ctx.repo_path)
+                dep_analyzer = build_tool.get_dep_analyzer(main_ctx.component.repository.fs_path)
             except DependencyAnalyzerError as error:
-                logger.error("Unable to find a dependency analyzer for %s: %s", tool.name, error)
-                continue
+                logger.error("Unable to find a dependency analyzer: %s", error)
```

Comment:
Is it intended to remove the name of the build tool in the error log message?

Reply:
No, see commit [5d52bef](https://github.com/oracle/macaron/pull/305/commits/5d52befa96f3514140ab90fe6b8d355f6e478e3a)